validation.Rmd

```diff
@@ -49,11 +50,28 @@ datasets are available, may be the best approach to model evaluation.
 This is the case with the CakeMap example explored in the previous chapter.
 There are no readily available datasets on the geographic distribution of cake consumption,
 so external validation of the dependent variable (frequency with which cake is
-eaten) is deemed impossiblen in this case. (However, new sources of data such as
+eaten) is deemed impossible in this case. (However, new sources of data such as
 number of confectionery shops, consumer surveys and even social media could be
 explored used to provide 'sanity checks' on the results. Sometimes you may need
 to be creative to find data for external validation.)
 
+An important remark on these two kinds of validation is taht it does not
+test the model in the same "level". Indeed, internal validation tests
+the model quality, considering the assumptions and the input data as 
+pertinent, correct and representative. If this validation fails, you may
+have a problem in the choice made during the modelisation, the algorithm, or 
```

Comment:
Modelisation is not an English word, although I like the sound of the French versions: http://fr.wikipedia.org/wiki/Mod%C3%A9lisation